From 79d4ef741fac65008505efe68ff2b3e1a64e8351 Mon Sep 17 00:00:00 2001
From: Zhao Qiang <qiang.zhao@nxp.com>
Date: Wed, 16 May 2018 14:45:33 +0800
Subject: [PATCH 478/767] iommu: fix compile issue

mc.h changes path, include the correct path

Signed-off-by: Zhao Qiang <qiang.zhao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/iommu/arm-smmu.c | 2 +-
 drivers/iommu/iommu.c    | 2 +-
 drivers/iommu/of_iommu.c | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/iommu/arm-smmu.c b/drivers/iommu/arm-smmu.c
index ffa3d6df046c..46da87bdc9d0 100644
--- a/drivers/iommu/arm-smmu.c
+++ b/drivers/iommu/arm-smmu.c
@@ -52,7 +52,7 @@
 #include <linux/spinlock.h>
 
 #include <linux/amba/bus.h>
-#include "../staging/fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 
 #include "io-pgtable.h"
 #include "arm-smmu-regs.h"
diff --git a/drivers/iommu/iommu.c b/drivers/iommu/iommu.c
index 21d3603533e7..56fccbf0a94d 100644
--- a/drivers/iommu/iommu.c
+++ b/drivers/iommu/iommu.c
@@ -33,7 +33,7 @@
 #include <linux/bitops.h>
 #include <linux/property.h>
 #include <trace/events/iommu.h>
-#include "../staging/fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 
 static struct kset *iommu_group_kset;
 static DEFINE_IDA(iommu_group_ida);
diff --git a/drivers/iommu/of_iommu.c b/drivers/iommu/of_iommu.c
index 079faee7c4f4..a9c77c6ac378 100644
--- a/drivers/iommu/of_iommu.c
+++ b/drivers/iommu/of_iommu.c
@@ -24,7 +24,7 @@
 #include <linux/of_iommu.h>
 #include <linux/of_pci.h>
 #include <linux/slab.h>
-#include "../staging/fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 
 #define NO_IOMMU	1
 
-- 
2.17.0

